### ISSUE TITLE
Advanced Search (old Complex Search) - Attributes do not match Attribute Search

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/hpi#930]
This could be a configuration problem, but if it is, I'm not seeing it.  On edge2's Contract Management trac, I'm seeing this:

*Contract Document - a few attributes configured and displaying in Attribute Search*
![image](https://cloud.githubusercontent.com/assets/1377228/11871946/60b71702-a498-11e5-9372-6a72d0f96a55.png)

*Contract Document - Advanced Search only displays Name*
![image](https://cloud.githubusercontent.com/assets/1377228/11871962/755c4434-a498-11e5-89aa-3df7d66a864f.png)

Shouldn't the attributes in Advanced Search mirror Attribute Search?

**Comment by gsteimer** [comment]
Quick update - in the console, I'm seeing this when I switch to Contract Document:

![image](https://cloud.githubusercontent.com/assets/1377228/11872271/14357c3c-a49a-11e5-8af6-17bd4dbcb2c0.png)

**Comment by gsteimer** [comment]
Update - from talking to @peterkk, I think this has something to do with proximity dates not being supported in the advanced search form.  Updating the title here - feel free to change it again if this turns out to not be the case.

**Comment by gsteimer** [comment]
Looking at this again - we should test Complex search with other form fields too.  The second field in the form is a Dropdown, which is rarely used.  I could also see Dropdown being an issue in Advanced search as well (maybe only Autocomplete was implemented?).  I've seen textbox, date, and autocomplete working.  we should also test:

- [ ] Checkbox
- [ ] Datetime
- [ ] DropDown
- [ ] ProximityDateSearch
- [ ] Radio
- [ ] Textarea (Don't worry about textarea list - it wouldn't be configured on a search form)

**Comment by dgrumieaux** [comment]
Waiting on code review according to @grahamsinger

**Comment by grahamsinger** [comment]
Yeah, this is ready to go in, but in fixing this bug @alieblich and I discovered a place in the advanced search code where the code could be improved in regards to how we are repopulating the models and views on a refresh of a page with search results/parameters.

Because of this I thought we would dicuss the best way to improve this part of the code going forward before we go ahead with the other unimplemented controls listed above.

**Comment by ZachGerth** [comment]
Added Proximity Date and Dropdown controls with commit # 14042. CR - grahamsinger

**Comment by ZachGerth** [comment]
Right now Checkbox, Datetime, Radio, and Textarea default to the Textbox control, and so don't really fit their control at all - these still need to be implemented for advanced search.
